package.json

```diff
@@ -14,47 +14,51 @@
   "dependencies": {
     "@babel/core": "^7.20.12",
     "@babel/eslint-parser": "^7.19.1",
-    "@next/eslint-plugin-next": "^13.1.1",
-    "@boxfish-studio/candymachine-client-sdk": "^0.0.4",
-    "@metaplex-foundation/js": "^0.11.7",
-    "@metaplex/arweave-cost": "^1.0.4",
+    "@boxfish-studio/candymachine-client-sdk": "^0.1.0",
+    "@metaplex-foundation/js": "^0.17.12",
+    "@next/eslint-plugin-next": "^13.1.5",
     "@primer/components": "^33.1.0",
-    "@primer/css": "^20.2.4",
-    "@primer/react": "^35.4.0",
-    "@project-serum/anchor": "^0.25.0",
-    "@solana/spl-token": "^0.2.0",
-    "@solana/wallet-adapter-base": "^0.9.5",
-    "@solana/wallet-adapter-react": "^0.15.4",
-    "@solana/wallet-adapter-react-ui": "^0.9.6",
-    "@solana/wallet-adapter-wallets": "^0.16.0",
-    "@solana/web3.js": "^1.41.3",
-    "@supercharge/promise-pool": "^2.1.0",
-    "@typescript-eslint/eslint-plugin": "^5.48.0",
+    "@primer/css": "^20.8.0",
+    "@primer/octicons-react": "^17.10.2",
+    "@primer/react": "^35.17.00",
+    "@project-serum/anchor": "^0.26.0",
+    "@solana/spl-token": "^0.3.7",
+    "@solana/wallet-adapter-base": "^0.9.20",
+    "@solana/wallet-adapter-react": "^0.15.28",
+    "@solana/wallet-adapter-react-ui": "^0.9.27",
+    "@solana/wallet-adapter-wallets": "^0.19.10",
```

Comment:
could this be updated?
![image](https://user-images.githubusercontent.com/72078237/214513649-0c80b8d1-3e72-4d2d-8804-acb69d7d13bc.png)